### PR TITLE
[WIP] Update RSA example

### PR DESCRIPTION
**Pull request by jaeilepp**
Take over from https://github.com/mne-tools/mne-python/pull/3552

It is using all 4 files. I removed the ``plot`` prefix, so it shouldn't run on circle.

### CODE REVIEW

**Comment by jaeilepp** [comment]
screenshot_2017-01-18_09-30-36
screenshot_2017-01-18_09-30-49

**Comment by kingjr** [comment]
ping me when ready for review

File: examples/decoding/plot_decoding_csp_eeg.py [hunk 1]
```diff
@@ -90,7 +90,7 @@
 # Use scikit-learn Pipeline with cross_val_score function
 from sklearn.pipeline import Pipeline  # noqa
 from sklearn.cross_validation import cross_val_score  # noqa
-clf = Pipeline([('CSP', csp), ('SVC', svc)])
+clf = Pipeline([('CSP', csp), ('SVC', lda)])
```
Comment (agramfort) [inline examples/decoding/plot_decoding_csp_eeg.py]: ('SVC', lda) -> ('LDA', lda)

File: examples/decoding/decoding_rsa.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+Representational Similarity Analysis is used to perform summary statistics
+on supervised classifications where the number of classes is relatively high.
+It consists in characterizing the structure of the confusion matrix to infer
+the similarity between brain responses and serve as a proxy for characterizing
+the space of mental representations [1-3].
+
+In this example, we perform RSA on spiking data from [4] where the macaque was
+presented to seven classes of pictures located in one of three spatial
+positions.
```
Comment (agramfort) [inline examples/decoding/decoding_rsa.py]: update this sentence. it's wrong now. We use the data from Cichy et al.

Comment (agramfort) [inline examples/decoding/decoding_rsa.py]: it's MEG data not spiking data.

**Comment by jaeilepp** [comment]
CIs are green now @kingjr

**Comment by Eric89GXL** [comment]
Is this MRG?

**Comment by jaeilepp** [comment]
From my part yes, but all I did here was to add the dataset. @kingjr gets the credit for the example.

**Comment by Eric89GXL** [comment]
Can you update the title then?

**Comment by agramfort** [comment]
I have a lot of local changes. I'll take over in the coming days. Please
don't merge.

**Comment by jaeilepp** [comment]
Ok. Changed it back to wip.

**Comment by kingjr** [comment]
I'll give a look after @agramfort s finished then

File: examples/decoding/decoding_rsa.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+from sklearn.model_selection import StratifiedKFold
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import roc_auc_score
+from sklearn.manifold import MDS
+
+from mne import find_events, Epochs, pick_types
+from mne.io import read_raw_fif, concatenate_raws
+from mne.datasets.visual_92_categories import data_path
```
Comment (jasmainak) [inline examples/decoding/decoding_rsa.py]: is this dataset documented in `doc/manual/datasets_index.rst`?

**Comment by agramfort** [comment]
closing in favor of #3923

**Pull request closed without merging**